src/firefly/js/Firefly.js

```diff
@@ -206,15 +214,18 @@ function installOptions(options) {
  * @param {AppProps} props
  * @param {FireflyOptions} options
  * @param {Array.<WebApiCommand>} webApiCommands
+ * @param {Array.<ClickToActionCommand>} clickToActionCommands
  */
-function fireflyInit(props, options={}, webApiCommands) {
+function fireflyInit(props, options={}, webApiCommands, clickToActionCommands) {
```

Comment:
These `webApiCommands, clickToActionCommands` are special where they cannot be in `options`?  My concern is the growing trend of these added features.

Reply:
oh I meant to take that parameter out. I was just experimenting. I put in options.